documentation/API_documentation/home_devices_qod_readiness_checklist.md

```diff
@@ -0,0 +1,13 @@
+# HomeDevicesQoD API Readiness minimum criteria checklist
+
+<br>
+
+| No | Deliverables/Criteria            | Mandatory | Status (Contributed/Approved/Pending/Validated/Partly-Validated)|
+|----|----------------------------------|-----------|----------------------------                         
+|  1 |API Spec                          |   Y       | Contributed                |
+|  2 |API Implementation                |   N       |                            |
+|  3 |API Documentation                 |   Y       | Contributed                |
+|  4 |User Stories                      |   Y       | Contributed                |
```

Comment:
Could links to User stories/test etc. be added in this column?

Reply:
Resolved in https://github.com/camaraproject/HomeDevicesQoD/pull/40/commits/27cb83819023aa630814b1a54a17955588d4c694